README.md

```diff
@@ -17,8 +17,8 @@ Learn more at [akka.io](https://akka.io/).
 Reference Documentation
 -----------------------
 
-The reference documentation is available at [doc.akka.io](http://doc.akka.io),
-for [Scala](http://doc.akka.io/docs/akka/current/scala.html) and [Java](http://doc.akka.io/docs/akka/current/java.html).
+The reference documentation is available at [doc.akka.io](https://doc.akka.io),
+for [Scala](https://doc.akka.io/docs/akka/current/scala.html) and [Java](https://doc.akka.io/docs/akka/current/java.html).
```

Comment:
Scala link redirects to `https://doc.akka.io/docs/akka/current/?language=scala`
Java link redirects to  `https://doc.akka.io/docs/akka/current/?language=java`

**Slash** before **question** in the URL. Is it ok?

Reply:
Yep, that's fine.